### ISSUE TITLE
Normalização para PEP8

### ISSUE THREAD

**Issue by vitornl** [vitornl/CheckErS2#17]
Documentação do código:
https://www.python.org/dev/peps/pep-0008/#comments

Exemplo prático:
https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/cluster/k_means_.py
